src/golang/executors/golang-with-dbs.yml

```diff
@@ -1,13 +1,13 @@
 description: Executor with the circleci/golang docker image as the default, plus postgres and redis
 docker:
-  - image: circleci/golang:<< parameters.tag >>
+  - image: cimg/go:<< parameters.tag >>
   - image: postgres:<< parameters.postgres-tag >>
   - image: redis:<< parameters.redis-tag >>
 parameters:
   tag:
-    description: tag of the circleci/golang docker image to use (Defaults to 'latest')
+    description: tag of the circleci/golang docker image to use (Defaults to '1.14')
     type: string
-    default: latest
+    default: "1.14"
```

Comment:
Do wonder if this should be required so it's not forgotten as part of version bumps.

Reply:
probably.